src/shared/Swagger/decorators/savedjobs/create-savedjobs.swagger.decorator.ts

```
@@ -0,0 +1,22 @@
+import { applyDecorators, HttpStatus } from "@nestjs/common";
+import { ApiOperation, ApiResponse } from "@nestjs/swagger";
+import { NotFoundSwagger } from "../../not-found.swagger";
+import { BadRequestSwagger } from "../../bad-request.swagger";
+
+export function SwaggerCreateSavedJobs() {
+  return applyDecorators(
+    ApiResponse({
+      status: HttpStatus.CREATED,
+      description: 'Cadastro realizado com sucesso. Retorna os dados do novo registro.',
+      type: NotFoundSwagger,
```

ESLint

[error] Delete `␍`

(prettier/prettier)